spec/controllers/videos_controller_spec.rb

```diff
@@ -4,41 +4,41 @@
   it {is_expected.to use_before_action(:require_user)}
   
   describe "GET show" do
+    before do
+      set_current_user
+      @video = Fabricate(:video)
```

Comment:
You should use `let` rather than instance variables. Instance variable state can persist across specs in some situations. `let` is the same as memoization using an instance variable anyway behind the scenes.